clusters/app.ci/ci-operator/admin_image-pusher_rbac.yaml

```diff
@@ -0,0 +1,22 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    openshift.io/description: This SA is used for pushing images to app.ci on build cluster
```

Comment:
Including the cluster here is kinda redundant, thats implicit by the cluster its on

Reply:
Updated.